enterprise/frontend/src/embedding-sdk/lib/plugins/initialize.ts

```diff
@@ -0,0 +1,5 @@
+import { flow } from "lodash";
```

Comment:
This accidentally works because there are some libraries that use `lodash`, but we don't pin that in our `package.json`. Our main util library is `underscore`. And we only pin selected lodash functions.

https://github.com/metabase/metabase/blob/59199fdd5a368e2662feea6a3b85758f6d03ac2d/package.json#L69-L70

Also, I don't think we need `flow` yet. Given we don't pass more than 1 function to it. We can just reexport `initializeDashboardPlugin` as `initialzePlugin` to get the same result.

On a side note, if you really need to do something like this, I'd suggest using `_.compose` which is a more standard way of combining functions.

Edit: I read the part where `initializePlugins` is used. I think we should export `initializeDashboardPlugins` directly, that way, if we really need to export `initializeQuestionPlugins` we could also do the same thing and we call these functions in different providers anyway.

Reply:
gotcha. that trips me up sometimes, nice catch